apps/cmp.c

```diff
@@ -1918,15 +1918,19 @@ static int setup_client_ctx(OSSL_CMP_CTX *ctx, ENGINE *engine)
                 goto err;
             }
         }
+
+        /* Will no more be needed after fixing callback design flaw #17088 */
```

Comment:
These comments throughout detract from the code and don't belong IMHO.

Reply:
Good point; I've pushed a fixup removing all five of them.